cypress/support/a11y/checkAxe.ts

```diff
@@ -21,20 +38,23 @@ function logViolations(violations: Result[]) {
     } ${violations.length === 1 ? 'was' : 'were'} detected.`
   );
 
-  // Destructure keys from the violations object to create a readable table
-  const violationData = violations.map(({ id, description, impact, nodes }) => ({
-      id,
-      description,
-      impact,
-      nodes: nodes.length
-    })
-  );
-
   // Print the table of violations to the console
   cy.task('table', violationData);
 }
 
-Cypress.Commands.add('checkAxe', (context, axeConfig, callback) => {
+function logViolationsAndThrow(violations: Result[]) {
+  _handleViolations(violations);
+}
+
+function logViolationsToConsoleOnly(violations: Result[]) {
+  _handleViolations(violations, true);
+}
+
+Cypress.Commands.add('checkAxe', (skipFailures, context, axeConfig, callback) => {
   cy.injectAxe();
-  cy.checkA11y(context ?? defaultContext, axeConfig ?? defaultAxeConfig, callback ?? logViolations);
+  if (skipFailures) {
+    cy.checkA11y(context ?? defaultContext, axeConfig ?? defaultAxeConfig, callback ?? logViolationsToConsoleOnly, skipFailures);
+  } else {
+    cy.checkA11y(context ?? defaultContext, axeConfig ?? defaultAxeConfig, callback ?? logViolationsAndThrow);
+  }
```

Comment:
I find this a bit repetitive, I'd personally lean towards a ternary for readability but that's just me
```suggestion
  cy.checkA11y(
    context ?? defaultContext,
    axeConfig ?? defaultAxeConfig,
    callback ?? skipFailures
      ? logViolationsToConsoleOnly
      : logViolationsAndThrow,
    skipFailures
  );
```